languages/julia/exercises/concept/multiple-dispatch/.meta/config.json

```diff
@@ -1,9 +1,10 @@
 {
-    "contributors": [
+    "authors": [
         {
             "github_username": "SaschaMann",
             "exercism_username": "SaschaMann"
         }
     ],
-    "language_versions": "≥1.0"
+    "language_versions": "≥1.0",
+    "forked_from": ["fsharp/basics"]
```

Comment:
Oh just noticed this, but isn't the other exercise forked from fsharp/basics? Doh.

Reply:
Indeed 🤦‍♂️